app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  def index
+    # ! @user = User.find(params[:id])
+    # ! @posts = Post.where(user_id: @user.id)
+  end
+
+  def show
+    # ! @user = User.find(params[:id])
+    # ! @post = @user.posts.find(params[:post_id])
```

Comment:
- [ ] @admirerbrown it's not good practice to have commented code in your codebase, kindly remove all the commented code.